community/resources/best-practices.md

```diff
@@ -0,0 +1,41 @@
+---
+group: community
+title: Community Contributed Best Practices
+---
+
+This section covers a broad scope of Community contributed best and leading practices, common solutions, and recommendations for Magento. 
+
+Topics and areas include:
+
+* Project Setup
+* Theme development (when and were to make a new theme)
+* Composer use (when to create a new repo)
+* Local environment options
+
+<div class="bs-callout bs-callout-info" markdown="1">
+**Note:** Magento does not officially endorse these best practices. 
+They are available in DevDocs because the content has been well received within the Magento community.
+</div>
+
+## Magento approved best practices
+
+The DevDocs and Magento teams verified and provide the following best practices and recommendations:
+
+- [Best Practices and Benchmarking]({{ page.baseurl }}/migration/migration-overview-practices.html) for Migration
+- [Best Practices for Extension Development]({{ page.baseurl }}/ext-best-practices/bk-ext-best-practices.html) 
+    
+    We also recommend the [Magento Marketplace Help Center](https://marketplacesupport.magento.com/hc/en-us) for extension questions.
+- [Programming Best Practices]({{ page.baseurl }}/ext-best-practices/extension-coding/common-programming-bp.html)
+- [Observers Best Practices]({{ page.baseurl }}/ext-best-practices/extension-coding/observers-bp.html)
+- [Theme development best practices]({{ page.baseurl }}/frontend-dev-guide/theme-best-practice.html)
+- [Performance Best Practices]({{ page.baseurl }}/performance-best-practices/)
+- [Best Practices for Store Configuration]({{ page.baseurl }}/cloud/configure/configure-best-practices.html) for Commerce Cloud
+- [Deployment process]({{ page.baseurl }}/cloud/reference/discover-deploy.html) for Commerce Cloud
+
+## Want to contribute?
+
+You can [contribute](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) by adding content and new topics through Pull Requests to the [DevDocs](https://github.com/magento/devdocs) repo. Save all files to the `/community/resources` directory. 
+
+We can help add navigation and format content when reviewing your submitted PRs. 
+
+If you have any questions, contact us through [#DevDocs Slack](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) (or [join us](https://t.co/9HImUyCmyh)) or Twitter [@MagentoDevDocs](https://twitter.com/MagentoDevDocs).
```

Comment:
The DevDocs Slack link goes to CONTRIBUTING.md

Reply:
Oops thanks! fixing